rust/candid/src/de.rs

```diff
@@ -135,6 +137,12 @@ macro_rules! check {
     }};
 }
 
+thread_local! {
```

Comment:
I'm not sure I understand why it has to be a thread-local. Can't we compute this array at the call site?